### PR TITLE
refactor(ruby): Use format strings

### PR DESCRIPTION
**Pull request by jletey**
#### Description

This PR deprecates `symbol` `style` keys in `RubyConfig`, and adds `format` key instead.

Based off of #1063.

#### Motivation and Context

Related #1057

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Screenshots (if appropriate):

Screenshot 2020-04-08 at 19 04 34

#### How Has This Been Tested?

- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -1142,19 +1142,24 @@ The module will be shown if any of the following conditions are met:
...
 
-| Variable   | Default      | Description                                            |
-| ---------- | ------------ | ------------------------------------------------------ |
-| `symbol`   | `"💎 "`      | The symbol used before displaying the version of Ruby. |
-| `style`    | `"bold red"` | The style for the module.                              |
-| `disabled` | `false`      | Disables the `ruby` module.                            |
+| Option     | Default                          | Description                 |
+| ---------- | -------------------------------- | --------------------------- |
+| `format`   | `"via [💎 $version](red bold) "` | The format for the module.  |
+| `disabled` | `false`                          | Disables the `ruby` module. |
```
Comment (andytom) [inline docs/config/README.md:1148]: We are keeping the `symbol` and `style` options to make the migration easier can you add them back please.

**Comment by jletey** [comment]
@andytom I have updated the PR, should be ready to go 👍

File: docs/config/README.md [hunk 2]
```diff
@@ -1465,13 +1465,22 @@ The module will be shown if any of the following conditions are met:
 - The current directory contains a `.ruby-version` file
 - The current directory contains a `.rb` file
 
-### Options
```
Comment (andytom) [inline docs/config/README.md:1468]: I think it is worth keeping this line to keep this inline with the other modules.

Comment (jletey) [reply docs/config/README.md:1468]: @andytom Whoops didn't even notice this 😅 Added it back :)

**Comment by andytom** [comment]
@jletey thanks for the quick fix.

**Pull request merged**